src/models/users.model.ts

```diff
@@ -4,6 +4,10 @@ import { Sequelize, DataTypes, Model } from 'sequelize';
 import { Application } from '../declarations';
 import { HookReturn } from 'sequelize/types/lib/hooks';
 
+export enum UserPermissions {
```

Comment:
That's not permissions that's `Roles`.

Reply:
changed bd58a083101312d0e3c7512ef88955c17798aeec